packages/nimble-components/src/table/models/tests/table-validator.spec.ts

```diff
@@ -238,6 +244,82 @@ describe('TableValidator', () => {
         });
     });
 
+    describe('column config validation', () => {
+        const columnConfigurations: {
```

Comment:
I'd like to see tests that exercise the default behavior (kinda what's shown here but it will be more explicit onse there is an actual class for that) and the expected usage by making an example class with an actual validation property in a simple example column that updates it's validation state.

Reply:
I introduced a test column type (and column validator type) in `table-column.fixture.ts` that I'm now using in both the table column tests and the table tests. Those tests should exercise the propagation of validation state.